### DISCUSSION TITLE
Heya! How do I use this extension with my extension?

### DISCUSSION THREAD

**Discussion by ShadiestGoat** [alefragnani/vscode-project-manager#452] [Q&A]
Heya! I am a developer, and I am getting into vscode extensions. Is there some sort of api from this extension that I can use? All I really need to do is to get the currently open project, but it would be nice if there was some sort of documentation for this stuff ^^

**Comment by alefragnani** [comment]
Hi @ShadiestGoat ,

Right now, the extension doesn’t expose any API. Sometime ago, another user asked for an _integration_ with the another extension (#365), but as I was focused on other ares, nothing was developed, yet.

But, 2021 is coming, and new features could be added 😬 .

Which APIs, besides _current project_, would you like to have available?
